### DISCUSSION TITLE
Discord Link Invalid

### DISCUSSION THREAD

**Discussion by singatias** [ctrlplanedev/ctrlplane#127] [General]
Hi there,

I'm interested to join the discord but the link is invalid!

Best Regards,
Singatias

**Comment by jsbroks** [comment]
Issue should be resolved: https://ctrlplane.dev/discord. Sorry about that! See you there

**Reply by singatias** [reply]
> Issue should be resolved: https://ctrlplane.dev/discord. Sorry about that! See you there

Just tried it and it still doesn't work!

![680d5e88ff9e5359409a43dd461162ae](https://github.com/user-attachments/assets/2ebbdec8-87d4-46a1-9b20-6386dd42cfea)

edit: nvm it works :)
